core/next.config.ts

```diff
@@ -38,7 +39,10 @@ async function writeSettingsToBuildConfig() {
     locales: data.site.settings?.locales,
     urls: {
       ...data.site.settings?.url,
-      cdnUrl: process.env.NEXT_PUBLIC_BIGCOMMERCE_CDN_HOSTNAME ?? data.site.settings?.url.cdnUrl,
+      cdnUrls: (process.env.NEXT_PUBLIC_BIGCOMMERCE_CDN_HOSTNAME
+        ? process.env.NEXT_PUBLIC_BIGCOMMERCE_CDN_HOSTNAME.split(',').map((s) => s.trim())
+        : [data.site.settings?.url.cdnUrl]
+      ).filter((url): url is string => !!url),
```

Comment:
This line here means that if `settings` is null we could get an empty array for URLs so probably a good idea to handle that case?

Reply:
Yes but I think that is fine. If all are empty (which is rare), I think it is fine that the server errors.

Reply:
But the error would happen somewhere else, in a very confusing way, instead of us explicitly handling the error here with a clear error message. 

If the expectation is we have the URL present, then we should enforce it instead of letting the server error later because the value is missing

Reply:
Got it! I added an error when there are no CDNs set.